src/Game.js

```diff
@@ -0,0 +1,46 @@
+import { Console, Random } from '@woowacourse/mission-utils';
+import { GAME, CRITERIA_NUMBER } from './constants/index.js';
+
+export class Game {
+  constructor(record, count) {
+    this.record = record;
+    this.count = count;
+  }
+
+  play() {
+    Console.print(GAME.START_MESSAGE);
+    for (let i = 0; i < this.count; i++) {
+      this.simulateRound();
+    }
+  }
+
+  simulateRound() {
+    const movedCars = this.getMovedCars();
+    this.record.update(movedCars);
+    this.print();
+  }
+
+  getMovedCars() {
+    return Object.keys(this.record.table)
+      .filter(this.shoudCarMove)
+      .map((car) => car);
+  }
+
+  shoudCarMove() {
+    return Random.pickNumberInRange(0, 9) >= CRITERIA_NUMBER;
+  }
+
```

Comment:
이 부분에선 오탈자가 있네요
`should`에서 l이 빠져있습니당
 
```suggestion
  shouldCarMove() {
    return Random.pickNumberInRange(0, 9) >= CRITERIA_NUMBER;
  }
  ```

Reply:
오탈자가 있었네요..1주차에도 오탈자가 있었는데 더 주의해야겠네요 ㅠㅠ